spec/features/stripe_customer_spec.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
```

Comment:
A lot of code from this file looks to be the same as the one from [stripe_checkout.spec](https://github.com/solidusio/solidus_stripe/blob/master/spec/features/stripe_checkout_spec.rb). For example, the `before` block is the same. I think it would be nice if we could avoid so much duplication. What about using shared examples, if feasible, or/and shared helper methods so we can DRY-up the files?

Reply:
Not a problem to DRY up the `before` block especially. In https://github.com/solidusio/solidus_stripe/pull/78 I was asked to put a helper into `lib/solidus_stripe/testing_support`. Do you want something there as well, or would you want it to live in the `spec/` path?

Reply:
Update! Was able to remove 172 lines of code from the spec by more aggressively using the `before` block and `shared_examples`. :)